### PR TITLE
monkey patch wagtail.contrib.redirects._get_redirect

### PR DESCRIPTION
**Pull request by Pomax**
At least, pending a PR landing somewhere. Probably wagtail-modeltranslation, but possible wagtail itself.

Addresses the core problem currently associated with https://github.com/mozilla/foundation.mozilla.org/issues/5634 and https://github.com/mozilla/foundation.mozilla.org/issues/5197

While the dedicated test command is:
```
inv manage "test networkapi.wagtailcustomization.tests.LocalizedRedirectTests"
```

_I cannot get tests to work_ so I've ended up posting a question to Stackoverflow about this because it's just baffling. However, local testing shows this to work exactly as desired.

### CODE REVIEW

File: network-api/networkapi/wagtailcustomization/tests.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+class LocalizedRedirectTests(TestCase):
+    def test_plain_redirect(self):
+        """
+        Test that the base redirect works.
+        """
+        Redirect.objects.create(
+            old_path='/test',
+            redirect_link='http://example.com'
+        )
+        response = self.client.get('/test')
```
Comment (Pomax) [inline network-api/networkapi/wagtailcustomization/tests.py:14]: This doesn't work in so many ways I posted https://stackoverflow.com/questions/65082910/how-to-create-a-wagtail-redirect-during-testing to hopefully understand why.

File: network-api/networkapi/wagtailcustomization/tests.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+from wagtail.contrib.redirects.models import Redirect
+
+
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailcustomization/tests.py:4]:
```suggestion
@override_settings(STATICFILES_STORAGE="django.contrib.staticfiles.storage.StaticFilesStorage")
```

File: network-api/networkapi/wagtailcustomization/tests.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    def test_plain_redirect(self):
+        """
+        Test that the base redirect works.
+        """
+        Redirect.objects.create(
+            old_path='/test',
+            redirect_link='http://example.com'
+        )
+        response = self.client.get('/test')
+        print(response)
```
Comment (KalobTaulien) [inline network-api/networkapi/wagtailcustomization/tests.py:15]:
```suggestion
        self.assertRedirects(response, 'http://example.com', status_code=301, fetch_redirect_response=False)
```

**Review by KalobTaulien — CHANGES_REQUESTED** [review]

File: env.default [hunk 4]
```diff
@@ -12,7 +12,7 @@ ASSET_DOMAIN=network.mofoprod.net
 CONTENT_TYPE_NO_SNIFF=True
 CORS_ALLOWED_ORIGINS=*
 CORS_ALLOWED_ORIGIN_REGEXES=
-DATABASE_URL=
+DATABASE_URL=postgresql://foundation@postgres:5432/wagtail
```
Comment (Pomax) [inline env.default:15]: I'm slipping this in, here, because this needs to be set.

**Review by KalobTaulien — APPROVED** [review]

**Pull request merged**